deploy/sre-build-test/50-source-build-test.CronJob.yaml

```diff
@@ -81,7 +82,18 @@ spec:
               oc create ns "${NS}"
 
               # run build
-              oc -n "${NS}" new-build nodejs~https://github.com/openshift/nodejs-ex --name="${JOB_PREFIX}"
+              oc -n "${NS}" new-build --name="${JOB_PREFIX}" --binary --strategy source --image-stream openshift/golang
+              cat <<EOF > /tmp/main.go
+              package main
+              import (
+                     "fmt"
+              )
+
+              func main() {
+                      fmt.Println("Hello Openshift SRE :)")
+              }
+              EOF
+              oc -n "${NS}" start-build "${JOB_PREFIX}" --from-file=/tmp/main.go
```

Comment:
wouldn't it be good enough to do `oc start-build -w -n ...` instead of the loop?

Reply:
instead of the `while` loop? Then we would have to assure the `-w` never drops, in which case we would wrap `-w` with `while true; do ..... -w;done` ?